File: 2024_04/0415/beginner/indexChange.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string my_string, int num1, int num2) {
    vector<char> char_vector(my_string.begin(), my_string.end());

    char temp = char_vector[num1];
    char_vector[num1] = char_vector[num2];
    char_vector[num2] = temp;
    return string(char_vector.begin(), char_vector.end());
}

File: 2024_04/0415/beginner/indexChange.js
function solution(my_string, num1, num2) {
    return my_string.substring(0,num1) + my_string.charAt(num2) + my_string.substring(num1 + 1, num2) + my_string.charAt(num1) + my_string.substring(num2+1);
}
